packages/fixed-decimal/src/FixedDecimal.ts

```diff
@@ -287,7 +287,7 @@ export class FixedDecimal<
     }
     const divisor = BigInt(10 ** digitsToRemove)
     const signOffset =
```

Comment:
I think this code would be a little easier to read with some more explicit early returns

```
if (digitsToRemove < 0) {
  throw new RangeError('Digits must be non-negative')
}
const divisor = BigInt(10 ** digitsToRemove)
if (this.value % divisor === BigInt(0)) {
  return new FixedDecimal({
    value: this.value,
    decimalPlaces: this.decimalPlaces
  })
}

if (this.value < 0) {
  return new FixedDecimal({
    value: (this.value / divisor) * divisor + BigInt(-1 * 10 ** digitsToRemove),
    decimalPlaces: this.decimalPlaces
  })
}

return new FixedDecimal({
    value: (this.value / divisor) * divisor,
    decimalPlaces: this.decimalPlaces
  })
}


```